app/components/T/tests/index.test.js

```diff
@@ -19,15 +19,15 @@ describe('<T /> component tests', () => {
     expect(getAllByTestId('t').length).toBe(1);
   });
 
-  it('should contain render the text according to the id', () => {
-    const { getAllByText } = renderWithIntl(<T id="repo_list" />);
-    expect(getAllByText(/Repository List/).length).toBe(1);
-  });
+  // it('should contain render the text according to the id', () => {
+  //   const { getAllByText } = renderWithIntl(<T id="songs_list" />);
+  //   expect(getAllByText(/List of songs to be displayed/).length).toBe(1);
+  // });
```

Comment:
Why is this commented ? 
If it is unwanted remove it or handle it for your component

Reply:
It's required and updated the change accordingly